src/executor.rs

```diff
@@ -11,13 +11,13 @@
 //!
 //! Run a single-threaded and a multi-threaded executor at the same time:
```

Comment:
for next time => if you are making changes close to descriptions like this, it's a good time to update the descriptions (I should have done it)

there is no multi-threaded executor, only async-executor had one.

This won't block the merge.

Reply:
I tool e4955970a4a2b85427f537df42d595d7a79082bc as an opportunity to update the documentation to:
```
//! # Examples
//!
//! Run four single-threaded executors concurrently:
//!
//! ```no_run
//! use scipio::{LocalExecutor, Timer};
//!
//! for i in 0..4 {
//!     std::thread::spawn(move || {
//!         let local_ex = LocalExecutor::new(None).expect("failed to create local executor");
//!         local_ex.run(async {
//!             Timer::new(std::time::Duration::from_millis(100)).await;
//!             println!("Hello world!");
//!         });
//!     });
//! }
//! ```
```

This is shorter and simpler to understand without knowledge of external crates.